### PR TITLE
Check out my Automated Torrent Setup, fixed.

### PR DESCRIPTION
**Pull request by ArionMiles**
pliss to merge.

### CODE REVIEW

File: data/cemeteries.json [hunk 1]
```diff
@@ -34,6 +34,15 @@
...
             }
+        },
+        
+        {
+            "username": "arion_miles",
+            "cemetery_data": {
+                "name": "Arion Miles",
+                "years_lived": "1992-2017",
+                "description": "<p>Here lies Arion Miles, in rest, finally. A no-name wannabe Pythonista.</p><p> His only contribution to this world was making a platform agnostic Automated Torrent Setup, about which he wouldn't shutup.</p><p>His last words: \"What's this on my dick?\".</p>"
+            }
```
Comment (avinassh) [inline data/cemeteries.json]: needs a comma. not a valid json. please check

Comment (ArionMiles) [inline data/cemeteries.json]: done, please review the changes.

Comment (avinassh) [inline data/cemeteries.json]: bru... I am sorry. looks like comma was not needed :|

Comment (ArionMiles) [inline data/cemeteries.json]: that's what I wondered. Because previous contributors didn't leave a comma. Do you want me to revert changes?

**Comment by avinassh** [comment]
you can add hyperlinks. Why not link it to your GH? check I have linked Rockstar on my cemetery page

**Pull request merged**